crates/router/src/connector/paypal.rs

```diff
@@ -1083,9 +1103,36 @@ impl api::IncomingWebhook for Paypal {
             )
             .into_report()
             .change_context(errors::ConnectorError::WebhookResourceObjectNotFound)?,
+            paypal::PaypalResource::PaypalDisputeWebhooks(_) => serde_json::to_value(details)
+                .into_report()
+                .change_context(errors::ConnectorError::WebhookResourceObjectNotFound)?,
         };
         Ok(sync_payload)
     }
+
+    fn get_dispute_details(
+        &self,
+        request: &api::IncomingWebhookRequestDetails<'_>,
+    ) -> CustomResult<api::disputes::DisputePayload, errors::ConnectorError> {
+        let payload: paypal::PaypalDisputeWebhooks = request
+            .body
+            .parse_struct("PaypalDisputeWebhooks")
+            .change_context(errors::ConnectorError::WebhookReferenceIdNotFound)?;
+        Ok(api::disputes::DisputePayload {
+            amount: payload.dispute_amount.value,
```

Comment:
our core expects amount in minor unit and PayPal process amount in base unit, can you confirm which unit we are getting in the web hooks